modules/builders/src/static-generator/index.ts

```diff
@@ -79,6 +85,8 @@ export async function execute(
         spinner.fail(`Prerendering routes to ${outputPath} failed.`);
 
         return { success: false, error: error.message };
+      } finally {
+        staticServer.close();
```

Comment:
Let's make sure to close the server prior to starting a new one. This is not an async call. 
Let's turn this into a promise.

```suggestion
       await staticServer.close();
```